.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+### Directions
+_GitHub Issues are reserved for actionable bug reports and feature requests._
+_General questions should be sent to the influxdb@googlegroups.com mailing list._
+
+_Before opening an issue, search for similar bug reports or feature requests on GitHub Issues._
+_If no similar issue can be found, fill out either the "Bug Report" or the "Feature Request" section below.
+_Erase the other section and everything on and above this line._
+
+### Bug report
+
+__System info:__ [Include InfluxDB version, operating system name, and other relevant details]
+
+__Steps to reproduce:__
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+__Expected behavior:__ [What you expected to happen]
+
+__Actual behavior:__ [What actually happened]
+
+__Additional info:__ [Such as links to config, log, or database files which exhibit the erroneous behavior]
+
+*Please note, the quickest way to fix a bug is to open a Pull Request.*
+
+
+### Feature Request
+
+Opening a feature request kicks off a discussion.
+Requests may be closed if we're not actively planning to work on them.
+
+__Proposal:__ [Description of the feature]
+
+__Current behavior:__ [What currently happens]
+
+__Desired behavior:__ [What you would like to happen]
+
+__Use case:__ [Why is this important (helps with prioritizing requests)]
+
+*Please note the quickest way to add a new feature is to open a Pull Request.*
```

Comment:
While this might be correct, something about this statement feels wrong.  Like I should feel guilty I didn't fix it.  Might just be me.  I'm fine with it as is, but just wanted to state that I'm not sure how I would feel about it when seeing it on a OSS project.  Maybe there is a better way to state our intention.

Something like:

We welcome all pull requests, as they result in a faster resolution.  We try to offer assistance on pull requests, so even if you aren't sure, feel free to attempt a fix.

Also, agree with @e-dard that we would prefer them to open an issue before they make major PR's.

I guess I've always felt I would rather see an issue with a proposed change prior to a PR (unless of course the fix is simple and obvious).